### PR TITLE
Fix http examples

### PR DESCRIPTION
**Pull request by abnegate**
## What does this PR do?

Fix HTTP examples
- File uploads
- Differentiate method `security` and method `auth` to allow correct `set` methods in SDK examples and correct headers in HTTP examples. Rename existing usages to `auth` to align with what is represented by the spec.

## Test Plan

Existing tests + manual docs checks

## Related PRs and Issues

https://github.com/appwrite/sdk-generator/pull/577

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/Spec/Swagger2.php [hunk 1]
```diff
@@ -169,7 +171,8 @@ public function getMethods($service)
                         'packaging' => $method['x-appwrite']['packaging'] ?? false,
                         'title' => $method['summary'] ?? '',
                         'description' => $method['description'] ?? '',
-                        'security' => [$auth] ?? [],
+                        'auth' => [$methodAuth] ?? [],
+                        'security' => [$methodSecurity] ?? [],
```
Comment (lohanidamodar) [inline src/Spec/Swagger2.php:175]: we are adding auth and security, are we using `security` for anything?

**Review by lohanidamodar — COMMENTED** [review]
Looks good, just a query.

File: src/Spec/Swagger2.php (hunk 1 above)
Comment (abnegate) [reply src/Spec/Swagger2.php:175]: To clarify: we already had `security`, but it was using `auth` from the spec. I have added `auth` and made this use `auth` from the spec, and changed the existing `security` to use `security` from the spec.

We are only using `security` in the HTTP examples here: https://github.com/appwrite/sdk-generator/blob/969bfa308f9504b7aaa45fa064f9e2ca83858343/templates/http/docs/example.md.twig#L10

Comment (lohanidamodar) [reply src/Spec/Swagger2.php:175]: Allright.

**Review by lohanidamodar — APPROVED** [review]
Looks good. Make sure we are not missing the `security` => `auth` change anywhere

**Pull request merged**